### PR TITLE
Fix recv()

### PR DESCRIPTION
**Pull request by damdam-s**

### CODE REVIEW

File: cubiscan/cubiscan.py [hunk 1]
```diff
@@ -14,11 +15,23 @@ class CubiScan(object):
...
-    def __init__(self, ip_address, port, timeout=30):
+    def __init__(self, ip_address, port, timeout=30, ssl=None,
+                 buffer_recv_size=1024):
         self.ip_address = ip_address
         self.port = port
         self.timeout = timeout
+        self.buffer_receive_size = buffer_recv_size
+
+        if not ssl:
+            self.ssl = False
```
Comment (guewen) [inline cubiscan/cubiscan.py]: I think `None` is a better value to keep in `self.ssl` when it's not set (I know we are used to have `False` everywhere with odoo ;)

**Review by guewen — APPROVED** [review]

**Comment by guewen** [comment]
Once merged, to have a new release, you'll only need to add a tag with the new lib's version, it will be pushed by travis to pypi.

**Comment by jgrandguillaume** [comment]
@damdam-s Can you change what's requested so we can proceed with the merge ?

File: cubiscan/cubiscan.py (hunk 1 above)
Comment (damdam-s) [reply cubiscan/cubiscan.py]: @guewen done

**Review by gurneyalex — APPROVED** [review]
LGTM

Don't forget to update HISTORY.rst when merging / preparing the new release.

**Pull request merged**